### PR TITLE
Add participants avatars bar

### PR DESCRIPTION
**Pull request by berkus**
Supercedes #107, see descriptions there.

### CODE REVIEW

File: src/components/conversations/ParticipantsView.tsx [hunk 1]
```diff
@@ -25,23 +50,58 @@ const ParticipantsView: React.FC<ParticipantsViewProps> = (
...
+  for (let i = 0; i < props.participants.length; i++) {
+    const participant = props.participants[i];
+    const name = fetchName(participant);
+    if (i < maxDisplayedParticipants) {
+      displayedParticipants.push(name);
+    } else {
+      hiddenParticipants.push(name);
+    }
+
+    if (i == MAX_HIDDEN_PARTICIPANTS - 1) {
```
Comment (rusmonster) [inline src/components/conversations/ParticipantsView.tsx:71]:
```
if (hiddenParticipants.length == MAX_HIDDEN_PARTICIPANTS) {
```

File: src/components/conversations/ParticipantsView.tsx [hunk 2]
```diff
@@ -25,23 +50,58 @@ const ParticipantsView: React.FC<ParticipantsViewProps> = (
...
+          >
+            <span
+              style={{
+                verticalAlign: "top",
+                paddingRight: 10,
+                color: theme.textColors.colorText,
+                fontWeight: theme.fontWeights.fontWeightSemibold,
+              }}
+            >
+              and {hiddenParticipants.length} other Participants
```
Comment (rusmonster) [inline src/components/conversations/ParticipantsView.tsx:100]: probably smth like
```
and {props.participants.length - displayedParticipants.length - hiddenParticipants.length}  other Participants
```
also display it only if the value > 0?

File: src/components/conversations/ParticipantsView.tsx [hunk 3]
```diff
@@ -1,12 +1,33 @@
...
+  maxDisplayedParticipants?: number;
+}
+
+function fetchName(participant: ReduxParticipant): string {
+  let name: string = participant.identity ?? "unknown";
+  if (participant.attributes != null) {
+    const friendlyName: string | null =
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      participant.attributes["friendlyName"];
```
Comment (rusmonster) [inline src/components/conversations/ParticipantsView.tsx:25]: who put it there? why it's not `User.friendlyName`?

File: src/components/conversations/ParticipantsView.tsx (hunk 1 above)
Comment (rusmonster) [reply src/components/conversations/ParticipantsView.tsx:71]: or rename `MAX_HIDDEN_PARTICIPANTS` -> `MAX_ALL_PARTICIPANTS` depending on what you mean here

File: src/components/conversations/ParticipantsView.tsx (hunk 2 above)
Comment (berkus) [reply src/components/conversations/ParticipantsView.tsx]: fixed ✅

File: src/components/conversations/ParticipantsView.tsx (hunk 1 above)
Comment (berkus) [reply src/components/conversations/ParticipantsView.tsx]: fixed ✅

**Review by rusmonster — APPROVED** [review]

**Pull request merged**